CMakeLists.txt

```diff
@@ -52,7 +52,8 @@ add_library(mender_compiler_flags INTERFACE)
 target_compile_features(mender_compiler_flags INTERFACE cxx_std_11)
 
 add_executable(mender main.cpp lib.cpp)
-target_link_libraries(mender PUBLIC mender_compiler_flags)
+target_link_libraries(mender PUBLIC common_events mender_compiler_flags)
```

Comment:
I was actually thinking we should nuke the toplevel main file.

Why keep it?

Reply:
I agree, but there's no rush. We can do that when we have the first pieces of an actual daemon. For now the `main.cpp` binary is still handy as a playground.